tests/system/data_sources/test_postgres.py

```diff
@@ -45,6 +45,28 @@
     "database": POSTGRES_DATABASE,
 }
 
+PG2PG_COLUMNS = [
+    "id" "col_int2",
```

Comment:
@nj1973 Is it intentional to have both strings in the same line? It seems to be the second should be placed in the next line preceded by a comma. But since the tests passed regardless, I got curious about this part.

Reply:
Good spot, that is a mistake, I have fixed it.